plugins/feeds/public/badip_apache_ddos_ip.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from datetime import timedelta
+
+from core.observables import Ip
+from core.feed import Feed
+from core.errors import ObservableValidationError
+
+# Time ranges for BadIp can be specified by y=year, day=d, minute=m, month=M
+class BadIPApacheDDOSIP(Feed):
+    default_values = {
+        'frequency': timedelta(hours=1),
+        'source': 'https://www.badips.com/get/list/apacheddos/0?age=1d',
```

Comment:
If you run this feed every hour (see L11), you might want to shorten the `?age=1d` parameter.